account_journal_general_sequence/models/account_journal.py

```diff
@@ -13,27 +13,20 @@ class AccountJournal(models.Model):
     entry_number_sequence_id = fields.Many2one(
         comodel_name="ir.sequence",
         string="Account entry number sequence",
-        default=lambda self: self._default_entry_number_sequence(),
+        compute="_compute_entry_number_sequence",
```

Comment:
IMHO you shouldn't have changed this default for a compute. The previous implementation was better because:

- It was just adding a default. If a user wanted to have a different sequence for another journal, they just had to change it. We'd be now missing this feature.
- It was making sure a sequence exists for the company when actually being used. See my comment on the `pre_init_companies` method for more details.